src/test/resources/isaac-test-es-docker-entrypoint.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -e
+
+# Files created by Elasticsearch should always be group writable too
+umask 0002
+
+
+### CUSTOM ISAAC PART
+# Remove any pre-existing index
+rm -rf /usr/share/elasticsearch/data
+# Expand our custom pre-built index
+tar -xpf isaac-test-es-data.tar.gz -C /usr/share/elasticsearch
+chown -R elasticsearch:root data
+rm isaac-test-es-data.tar.gz
+# Remove node 0's lock otherwise ES cannot start
+rm -f /usr/share/elasticsearch/data/nodes/0/node.lock
+# Also remove write locks for node 0 otherwise ES cannot start
+find /usr/share/elasticsearch/data -name write.lock -exec rm -f {} \;
+### END OF CUSTOM ISAAC PART
```

Comment:
Yes, I tried many alternatives but this has to be run between the container's creation and the container's ES startup. Unfortunately, there's no way I could find of injecting a command between those two moments, and it's also not feasible to make the changes, stop the container, then start it again, because testcontainers removes everything on `stop()` and there is no `restart()` (see [this discussion](https://github.com/testcontainers/testcontainers-java/issues/606)). Elasticsearch doesn't seem to have a way of soft-restarting itself via signal, anything I tried led to a sudden stop, and it doesn't even have a way of rereading the data files from scratch while running, so this is the best I could come up with.

On the plus side, I don't expect the startup script to change very much, so I guess we can live with a reminder to check this every time we upgrade testcontainers.